enhancements/network/baremetal-ipi-network-configuration.md

```diff
@@ -0,0 +1,281 @@
+---
+title: baremetal-ipi-network-configuration
+authors:
+- "@cybertron"
+- "@hardys"
+- "@zaneb"
+reviewers:
+- "@kirankt"
+- "@dtantsur"
+- "@zaneb"
+approvers:
+- "@trozet"
+creation-date: 2021-05-21
+last-updated: 2021-07-07
+status: provisional|implementable
+
+see-also:
+- "/enhancements/host-network-configuration.md"
+- "/enhancements/machine-config/mco-network-configuration.md"
+- "/enhancements/machine-config/rhcos/static-networking-enhancements.md"
+---
+
+# Baremetal IPI Network Configuration
+
+Describe user-facing API for day-1 network customizations in the IPI workflow,
+with particular focus on baremetal where such configuration is a common
+requirement.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Currently in the IPI flow, there is no way to provide day-1 network configuration
+which is a common requirement, particularly for baremetal users.  We can build
+on the [UPI static networking enhancements](rhcos/static-networking-enhancements.md)
```

Comment:
NIT. This relative URL doesn't work correctly. Suggestion: We can use: https://github.com/openshift/enhancements/blob/master/enhancements/rhcos/static-networking-enhancements.md